framework/src/org/checkerframework/framework/qual/AnnotatedFor.java

```diff
@@ -11,14 +14,14 @@
  * For example, <code>@AnnotatedFor({"nullness", "regex"})</code> indicates
  * that the class has been annotated with annotations such as
  * <tt>@Nullable</tt> and <tt>@Regex</tt>.  Has no effect unless the
- * <tt>-AuseSafeDefaultsForUnannotatedSourceCode</tt> command-line argument
+ * <tt>-AuseDefaultsForUncheckedCode=source</tt> command-line argument
  * is supplied.
  * <p>
  *
- * Ordinarily, the <tt>-AuseSafeDefaultsForUnannotatedSourceCode</tt> command-line argument
- * causes unannotated locations to be defaulted using conservative library
- * annotations, and it suppresses all warnings.  The
- * <tt>-AuseSafeDefaultsForUnannotatedSourceCode</tt> command-line argument has no effect on
+ * Ordinarily, the <tt>-AuseDefaultsForUncheckedCode=source</tt> command-line argument
+ * causes unannotated locations to be defaulted using unchecked code defaults,
+ * and it suppresses all warnings.  The
```

Comment:
I know you didn't write this, but: I think this would be clearer if "The" were changed to "However, the". That way "however" is paired with the "Ordinarily" at the beginning of the paragraph. Otherwise the "Ordinarily" is confusing - which circumstances are not considered ordinary?